pkg/quantile/agent.go

```diff
@@ -56,3 +56,17 @@ func (a *Agent) Insert(v float64) {
 
 	a.flush()
 }
+
+// InsertN inserts v, n times into the sketch.
+func (a *Agent) InsertN(v float64, n uint) {
+	a.Sketch.Basic.InsertN(v, n)
+
+	for i := 0; i < int(n); i++ {
+		a.Buf = append(a.Buf, agentConfig.key(v))
```

Comment:
This part would benefit from having bulk insert in the the sparse store https://github.com/DataDog/datadog-agent/blob/e96af32dcc13e674a2ac1ce8bec240c0e2bf4602/pkg/quantile/store.go#L133